### ISSUE TITLE
cifar 100 parameters

### ISSUE THREAD

**Issue by ielenik** [Rishit-dagli/Astroformer#1]
Can you please share training parameters for cifar-100 dataset?
Thank you

**Comment by HoldMyHande** [comment]
Have you got some ideas?I meet the same problem now,what is your accuracy on cifar-100?Have a nice day

**Comment by sjtuccz** [comment]
Training astroformer3 from scratch with cifar100 did not achieve the 87.65 accuracy rate mentioned in the article . I got a result by a wide margin 78.46. Moreover, adjusting the learning rate has little effect on the accuracy.

CUDA_VISIBLE_DEVICES=3 python train.py --dataset torch/cifar100 --data-dir pytorch-cifar100-master/data --model astroformer_3 --num-classes 100 --in-chans 3 --input-size 3 224 224 --batch-size 128 --grad-accum-steps 1 --opt adamw --sched cosine --lr-base 2e-5 --lr-cycle-decay 1e-2 --lr-k-decay 1 --warmup-lr 1e-5 --epochs 300 --warmup-epochs 5 --mixup 0.8 --smoothing 0.1 --drop 0.1 --amp --output /Astroformer-main/output

![Image](https://github.com/user-attachments/assets/f6e83a0f-cb96-42a5-b135-6995eebbe4e6)
